lib/miq_expression.rb

```diff
@@ -358,7 +358,7 @@ def load_virtual_custom_attributes
 
   def self.proto?
     return @proto if defined?(@proto)
-    @proto = VMDB::Config.new("vmdb").config.fetch_path(:product, :proto)
+    @proto = ::Settings.fetch_path(:product, :proto)
   end
```

Comment:
Dot notation should work here since ```product``` is required entry:
```
>> ::Settings.product.proto
=> nil
>> ::Settings.fetch_path(:product, :proto)
=> nil
>> ```